### ISSUE TITLE
Correct the checklists for when to check the fuel gauges

### ISSUE THREAD

**Issue by gilbertohasnofb** [Juanvvc/c172p-detailed#144]
Reported by bugman in the forum thread: 

> Just a quick one, someone should check the checklists for when to check the fuel gauges. Something is not quite right there.

**Comment by gilbertohasnofb** [comment]
bugman:
> The fuel gauges are not active at that point in the checklist sequence, so a newbie checking them might believe that they won't be able to fly very far.

ludomotico:
> You are right. The fuel level gauges are only active when master is ON. During the preflight inspection, you set master ON, check fuel level, flaps DOWN and immediately master OFF. The external inspection is done with flaps down. The whole pre-flight inspection was dismissed as a single line in the checklist because most of the action happens outside the aircraft and there is not many things to do in a simulator outside the aircraft (I think it was me who did this). But we still have an old "check fuel gauges" in the pre-flight list. So, we either restore the pre-flight inspection, or add a new item to the beginning of the list "master ON, check fuel level, master OFF".

bugman:
> For learning and young pilots it might be nice to have something guide them through all the paces, especially in the default FG aircraft :) From memory there was a 'check the fuel guages' in the c172p (checklist or tutorial - I just can't check right now). Anyway, just my 2 cents.

**Comment by gilbertohasnofb** [comment]
bugman:
> Both the fuel gauge and avionics power switch problems are in the "Before Starting Engine" checklist. As the avionics switch is off at the start, the checklist 'OFF' text should be green, however it starts as orange.

**Comment by onox** [comment]
I noticed a problem with the mater avionics switch behaving weird in the checklist. Does this patch fixes it?

```diff
diff --git a/c172p-detailed-set.xml b/c172p-detailed-set.xml
index 0e3c1fc..87de060 100644
--- a/c172p-detailed-set.xml
+++ b/c172p-detailed-set.xml
@@ -222,6 +222,9 @@ Many files from the original c172p directory are included in this file.
     <primer-lever>0</primer-lever>
    </engine>
   </engines>
+  <switches>
+    <master-avionics type="bool">false</master-avionics>
+  </switches>
  </controls>
 
  <autopilot>
```

**Comment by tigert** [comment]
I think this has been fixed recently, by 676e1c5d44fa046d39a3353f82a26eb3e7981acf

**Comment by onox** [comment]
@Juanvvc Can you verify if this issue has been fixed?
